ospf6d/ospf6_message.c

```diff
@@ -1532,10 +1532,15 @@ int ospf6_receive(struct thread *thread)
 	struct iovec iovector[2];
 	struct ospf6_interface *oi;
 	struct ospf6_header *oh;
+	struct ospf6 *ospf6;
 
 	/* add next read thread */
+	ospf6 = THREAD_ARG(thread);
 	sockfd = THREAD_FD(thread);
-	thread_add_read(master, ospf6_receive, NULL, sockfd, NULL);
+	ospf6->t_ospf6_receive = NULL;
```

Comment:
Nit: it is not required anymore to set the `struct thread *` to `NULL` `libfrr` already does that.

Reply:
Fixed